layers/gd/Dockerfile

```diff
@@ -0,0 +1,55 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION as ext
+
+# Install required build libraries
+RUN set -ue \
+    ; LD_LIBRARY_PATH= \
+    ; yum -y install \
+        libwebp-devel \
+        libXpm-devel \
+        libpng-devel \
+        libjpeg-devel \
+        freetype-devel \
+    ; yum clean all \
+;
+
+WORKDIR ${PHP_BUILD_DIR}/ext/gd
+RUN phpize
+ARG PHP_VERSION
+RUN if [ "$PHP_VERSION" = "73" ] ; \
```

Comment:
a bit ugly IMHO ... the condition should target only the different stuff .. never repeat the common part .. WYT?

Something like the following (not tested it could contains errors ...)
```
LIB_STR="
    --with-gd
    --with-jpeg
    --with-png
    --with-xpm
    --with-webp
    --with-zlib
" ; \
if [ ... ]; then
    LIBS_STR="
        --with-gd
        --with-jpeg-dir=${INSTALL_DIR}
        --with-png-dir=${INSTALL_DIR}
        --with-xpm-dir=${INSTALL_DIR}
        --with-webp-dir=${INSTALL_DIR}
        --with-zlib-dir=${INSTALL_DIR}
   " 
fi; \
./ configure \
        --disable-static \
        --enable-gd-jis-conv \
        --enable-shared \
;
```